File: tests/data-clumps/test-cases/field-field/simple/negative/java/source/Doctor.java
public class Doctor {
    public int doctorId;
    public boolean isOnCall;
    public int appointmentSlots;

    public Doctor(int doctorId, boolean isOnCall, int appointmentSlots) {
        this.doctorId = doctorId;
        this.isOnCall = isOnCall;
        this.appointmentSlots = appointmentSlots;
    }
}

File: tests/data-clumps/test-cases/field-field/simple/negative/typescript/source/Doctor.ts
export class Doctor {
  public doctorId: number;
  public isOnCall: boolean;
  public appointmentSlots: number;

  constructor(doctorId: number, isOnCall: boolean, appointmentSlots: number) {
    this.doctorId = doctorId;
    this.isOnCall = isOnCall;
    this.appointmentSlots = appointmentSlots;
  }
}
